src/Enum.php

```diff
@@ -118,8 +118,8 @@ public static function values()
      */
     public static function toArray()
     {
-        $class = get_called_class();
-        if (!array_key_exists($class, static::$cache)) {
+        $class = \get_called_class();
+        if (!isset(static::$cache[$class])) {
```

Comment:
Are you sure this change doesn't introduce BC breaks?

Reply:
Yes. `static::$cache` can only be instances of array, for which `isset` returns the exact values as `array_key_exists`. The only difference between the two is their behaviour around null values.